### DISCUSSION TITLE
Работа над Core

### DISCUSSION THREAD

**Discussion by Sskame1** [JustAlexeyDev/EvaOS#13] [Show and tell]
Я написал обновление Core где добавил достаточно много всего для работы с файловой системой.
p.s я не знаю почему у меня не хочет работать pull requests.

файл:
[Core.zip](https://github.com/user-attachments/files/16337354/Core.zip)

![image](https://github.com/user-attachments/assets/1081eab5-5293-4d78-b67d-b2473a45863c)

**Comment by JustAlexeyDev** [comment]
> Я написал обновление Core где добавил достаточно много всего для работы с файловой системой.
> p.s я не знаю почему у меня не хочет работать pull requests.
> 
> файл:
> [Core.zip](https://github.com/user-attachments/files/16337354/Core.zip)
> 
> ![image](https://github.com/user-attachments/assets/1081eab5-5293-4d78-b67d-b2473a45863c)
> 
> 
> 

Ждём дальнейшее развитие файловой системы!
